requirements-docs.txt

```diff
@@ -0,0 +1,2 @@
+Sphinx==1.1.3
+-rrequirements.txt
```

Comment:
It is so you can pull in the most recent botocore, which is needed for boto3 because the doc logic for botocore live in boto3. I would be fine with the ``-e`` option for the CLI because this logic is not really too coupled with botocore.